CHANGELOG/CHANGELOG-2.0.md

```diff
@@ -0,0 +1,89 @@
+# Changelog 2.0
+
+## 2.0.0 - 2024-04-26
+
+### Added
+
+- Support `LoadBalancerIPMode` in AntreaProxy to implement K8s [KEP-1860](https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/1860-kube-proxy-IP-node-binding). ([#6102](https://github.com/antrea-io/antrea/pull/6102), [@hongliangl])
+- Add `sameLabels` field support for Antrea ClusterNetworkPolicy peer Namespace selection to allow users to create ACNPs that isolate Namespaces based on their label values. ([#4537](https://github.com/antrea-io/antrea/pull/4537), [@Dyanngg])
+- Add multiple physical interfaces support for secondary network bridge. ([#5959](https://github.com/antrea-io/antrea/pull/5959), [@aroradaman])
+- Add the user document for Antrea native secondary network support. ([#6015](https://github.com/antrea-io/antrea/pull/6015) [#6042](https://github.com/antrea-io/antrea/pull/6042), [@jianjuns] [@antoninbas])
+- Add a new versioned API `NetworkPolicyEvaluation` and a new antctl sub-command for querying the effective policy rule applied to particular traffic. ([#5740](https://github.com/antrea-io/antrea/pull/5740) [#6112](https://github.com/antrea-io/antrea/pull/6112), [@qiyueyao])
+
+### Changed
+
+- Multiple deprecated APIs, fields and options are removed from Antrea since v2.0.0
+  - Remove a few deprecated v1alpha1 CRDs including `Tier`, `ClusterNetworkPolicy`, `NetworkPolicy`, `Traceflow` and `ExternalEntity`. ([#6162](https://github.com/antrea-io/antrea/pull/6162) [#6177](https://github.com/antrea-io/antrea/pull/6177) [#6238](https://github.com/antrea-io/antrea/pull/6238), [@luolanzone] [@hjiajing] [@antoninbas])
+  - Remove a few deprecated v1alpha2 and v1alpha3 CRDs including `ClusterGroups`, `ExternalIPPool`, `ClusterGroup` and `Group`. ([#6049](https://github.com/antrea-io/antrea/pull/6049) [#6239](https://github.com/antrea-io/antrea/pull/6239), [@luolanzone] [@antoninbas])
+  - Remove deprecated `ServiceAccount` field in ClusterSet type for Antrea Multi-cluster. ([#6134](https://github.com/antrea-io/antrea/pull/6134), [@luolanzone])
+  - Remove a few deprecated options including `enableIPSecTunnel`,`multicastInterfaces`, `multicluster.enable` and `legacyCRDMirroring`. ([#5158](https://github.com/antrea-io/antrea/pull/5158), [@luolanzone])
```

Comment:
Similar comment as above: we should have a comprehensive list of deprecated options here IMO, so the sentence should really read "Remove deprecated options [list]"